lib/Models/Internationalization.ts

```diff
@@ -48,11 +82,15 @@ class Internationalization {
       .use(initReactI18next)
       .init({
         debug: languageConfig.debug,
+        react: languageConfig.react,
         fallbackLng: languageConfig.fallbackLanguage,
         whitelist: Object.keys(languageConfig.languages),
 
         // to allow en-US when only en is on the whitelist - nonExplicitWhitelist must be set to true
-        nonExplicitWhitelist: true,
+        // nonExplicitWhitelist: true,
+        // deprecated, it's now the inverse
+        nonExplicitSupportedLngs: false,
```

Comment:
@zoran995 nonExplicitWhitelist is now deprecated, is it right to do `nonExplicitSupportedLngs` as `false` (which is the default anyway) to get the same behaviour?

Reply:
also `whitelist` [L87](https://github.com/TerriaJS/terriajs/pull/4613/files#diff-89ed42d66d62ba7736eb980b2e4778a7R87) was changed to `supportedLanguages`, there is no much documentation about migration to the new definition, going to test this today

Reply:
from my understanding it is not inverse, `nonExplicitWhitelist` was also default to `false`. On my end everything works fine with both settings.

Reply:
Yes you're right it should be `true` here - also in my mind I conflated the config surrounding this a little bit with `load: "languageOnly"` - but upon looking at the source in i18next this confirms that they both intend to do similar things:
```
    if (this.options.load === 'languageOnly' || this.options.nonExplicitSupportedLngs) {
      code = this.getLanguagePartFromCode(code);
    }
```
https://github.com/i18next/i18next/blob/80a38100d21a7e7c1f9cb2acff5f709063027b9f/src/LanguageUtils.js#L78-L80